platform/vs/docker-sonic-vs/Dockerfile.j2

```diff
@@ -147,6 +147,7 @@ COPY ["zero_profiles.json", "/etc/sonic"]
 COPY ["buffermgrd.sh", "/usr/bin/"]
 
 COPY ["platform.json", "/usr/share/sonic/device/x86_64-kvm_x86_64-r0/"]
+COPY ["platform-npu.json", "/usr/share/sonic/device/x86_64-kvm_x86_64-r0/"]
```

Comment:
can we change to platform-npu-2p.json, meaning this is 2 port npu, maybe in the future we will have 1 port or 4 port npu.

Reply:
ok, will do